test/ClangImporter/enum-anon-sized.swift

```diff
@@ -0,0 +1,31 @@
+// RUN: not %target-swift-frontend -typecheck %s -enable-objc-interop -import-objc-header %S/Inputs/enum-anon.h -DDIAGS 2>&1 | %FileCheck %s -check-prefix CHECK-DIAG
+// RUN: %target-swift-frontend -emit-ir %s -enable-objc-interop -import-objc-header %S/Inputs/enum-anon.h | %FileCheck -check-prefix=CHECK -check-prefix=CHECK-%target-runtime %s
+
+// We do not use `-verify` as the types are imported with differing types and we
+// cannot match loosely with `-verify`.
+
+#if DIAGS
+let _: String = Constant2
+// CHECK-DIAG: :8:17: error: cannot convert value of type 'Int' to specified type 'String'
+let _: String = VarConstant2
+// CHECK-DIAG: :10:17: error: cannot convert value of type '{{U?}}Int32' to specified type 'String'
+let _: String = SR2511().y
+// CHECK-DIAG: :12:26: error: cannot convert value of type '{{U?}}Int32' to specified type 'String'
```

Comment:
Is it possible to conditionalize this on Windows vs. non-Windows?

Reply:
I couldn't come up with one.  The problem is that I would normally use the `-verify` option to clang with the preprocessor ... but, I don't have that hammer here.  Can you think of some tricky swiftc usage to do something similar?

Reply:
All the diagnostic is doing is confirming that the type is the same; we know how to do that in-source with `func checkTypeIsInt32(_: inout Int32)`. So maybe we can conditionalize that on `os(Windows)`?

Reply:
Yeah, that should work ... and we can do that in a single shot, since the IRGen will fail if the verification fails.  Awesome!